### ISSUE TITLE
Warning marks in java script files works only after reopen file .

### ISSUE THREAD

**Issue by Ohrimenko1988** [eclipse-che/che#4954]
**Reproduction Steps:**
- Go to the spring project
- Create some javascript file
- Type :
              var a = 5;
              var c = 6;

             function f(){
                 return 'test'
             }

            f();
- Wait some times
- Pay attention to the screen.

**Expected behavior:**
Warnings marks will be displayed

**Observed behavior:**
Warnings marks are not display 

**Attachment:**

![anim](https://cloud.githubusercontent.com/assets/22724619/25527653/66c9346e-2c22-11e7-9b71-f23be68167db.gif)

**Comment by ghost** [comment]
Duplicate of https://github.com/eclipse/che/issues/4886
